sound/soc/sof/topology.c

```diff
@@ -898,17 +865,32 @@ static int sof_parse_word_tokens(struct snd_soc_component *scomp,
 	return found;
 }
 
-static int sof_parse_tokens(struct snd_soc_component *scomp,
-			    void *object,
-			    const struct sof_topology_token *tokens,
-			    int count,
-			    struct snd_soc_tplg_vendor_array *array,
-			    int priv_size)
-{
+/**
+ * sof_parse_token_sets - Parse multiple sets of tokens
+ * @object: target ipc struct for parsed values
+ * @tokens: token definition array describing what tokens to parse
+ * @count: amount of tokens in definition array
```

Comment:
I think "number of..." would be a bit clearer, also below for `sets`

Reply:
ok